packages/breadboard-ui/src/types/types.ts

```diff
@@ -94,3 +95,19 @@ export type OutputArgs = {
     schema?: Schema;
   } & Record<string, unknown>;
 };
+
+export const ErrorNames = {
```

Comment:
AFAICT we don't need these types as the information is already available on the Events themselves?